src/components/Square.js

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
-  //  component when it's clicked on.
-
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
-
+  const callback = () => {
+    props.onClickCallback(props.id);
+  };
+  return (
+    <button className="square" data-pro={props.value} onClick={callback}>
+      {props.value}
+    </button>
+  );
+};
```

Comment:
We could use object destructing so we can reference props by their key name instead.

```suggestion
const Square = ( {value, id, onClickCallback, key} ) => {
  const callback = () => {
    props.onClickCallback(id);
  };
  return (
    <button className="square" data-pro={value} onClick={callback}>
      {value}
    </button>
  );
};

```